burst/README.md

```diff
@@ -0,0 +1,87 @@
+# Burst Metrics Service
+
+Burst metrics service is responsible for reading HPA metrics from a Kubernetes(k8s) API and return a formatted burst metrics header. It is implemented as a REST service.
+
+Currently it will capture current pod count and target pod count from an HPA.
+
+## Deployment/usage
+
+TODO: add recently pushed `make` usage
+
+## API Endpoints
+
+### `/burstmetrics/{namespace}/{HPA}`
+
+This is the only endpoint exposed by this service.
+This endpoint will check the k8s `{namespace}` for the specified `{HPA}`.
+
+If the `{HPA}` is found and is accessible, it will return the metrics below in a formatted fashion:
+
+- service=namespace/HPA-name
+- current-load=current pod count
+- max-load=target hpa pod count
+- target-load=80% of max-load <!-- WIP:Configure the percentage -->
+
+If it can't find the `{HPA}` or is inaccessible, or any exception occurs during the API call, it will simply return `HTTP 204 No Content`.
+
+#### Example 1: 200 OK
+
+```http
+HTTP/1.1 200 OK
+content-type: text/plain; charset=utf-8
+date: Mon, 23 Aug 2021 16:29:51 GMT
+server: istio-envoy
+transfer-encoding: chunked
+x-envoy-upstream-service-time: 1
+
```

Comment:
do we need to run the service under Istio? There's an issue on the board to address. This will need to be updated.

Reply:
We do not need istio.
Added update documentation subtask in https://github.com/retaildevcrews/wcnp/issues/303